0.1.Copy data from Excel codes.R

```diff
@@ -1,79 +1,75 @@
-rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
-gc() #free up memrory and report the memory usage.
+#rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
+#gc() #free up memrory and report the memory usage.
 memory.limit(size = 80000000)
 
 library(xlsx)
 library(ggplot2)
 library(ggpmisc)
 library(readxl)
+library(dplyr)
+library(forcats)
+library(bayestestR)
 
+# set data locations in one place
+inputdata <- c(subjects="Subjects with Clamp Infor.xlsx"
+               ,dat0="Erit-2 Clamp Calculations Dr M with lean mass (003)nm updated with 206 215_2.xlsx");

```

Comment:
Just like with the libraries, the input files should be defined in one place, early in the script, for the same reasons. Just as in class, I'm creating a vector of file locations where each file location has a name reminding us what it's for. Further down you will see me refer to them by that name. So if you use the full paths to your real data while I use the facsimile copies in this project directory, only one line needs to be updated.